src/v/coproc/offset_storage_utils.cc

```diff
@@ -147,7 +85,17 @@ ss::future<> save_offsets(
       ntp_cache.size());
     /// Create the metadata, and data iobuffers
     iobuf metadata = reflection::to_iobuf(static_cast<int8_t>(1));
-    iobuf data = co_await serialize_data_field(ntp_cache);
+    iresults_map irm_map;
+    irm_map.reserve(ntp_cache.size());
+    std::transform(
+      ntp_cache.cbegin(),
+      ntp_cache.end(),
```

Comment:
make sure to batch `c`begin with `c`end. back in the day it wasn't always the case that the iterators were the same in some stdlib implementations, and I'm not sure it is actually guaranteed by the standard or not.